apps/site/lib/site_web/controllers/schedule/green.ex

```diff
@@ -259,13 +259,21 @@ defmodule SiteWeb.ScheduleController.Green do
   defp arrival_time(schedule), do: schedule.time
 
   defp validate_direction(
-         %{assigns: %{origin: origin, destination: destination, direction_id: direction_id}} =
-           conn,
+         %{
+           assigns: %{
+             origin: origin,
+             destination: destination,
+             direction_id: direction_id
+           }
+         } = conn,
          _
        )
        when not is_nil(origin) and not is_nil(destination) do
     {stops, _map} = conn.assigns.stops_on_routes
 
+    stops = Enum.reverse(stops)
+    Util.ListHelpers.find_first(stops, origin, destination)
```

Comment:
@phildarnowsky confused by this line, looks like this is being called but the return value discarded?

Reply:
@ryan-mahoney I probably had an `IO.inspect` in there, this line can go